Source/Core/Core/HW/EXI_DeviceIPL.cpp

```diff
@@ -304,7 +304,7 @@ void CEXIIPL::TransferByte(u8& _uByte)
 				{
 					u32 position = ((m_uAddress >> 6) & ROM_MASK) + m_uRWOffset;
 
-					// Technically we should descramble here iff descrambling logic is enabled.
+					// Technically we should descramble here only if descrambling logic is enabled.
```

Comment:
Just to link onto this "iff" means "if and only if" in terms of logic/mathematics, so it's technically not a typo here.